src/test/java/Login_SignUp_Test.java

```diff
@@ -0,0 +1,110 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+
+public class Login_SignUp_Test extends BaseTest {
+
+  private static By email = By.cssSelector("#email");
+  private static By pass = By.cssSelector("#password");
+  private static By username = By.cssSelector("#username");
+
+  private void clickLoginButton() {
+    WebElement loginButton = driverWait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("[data-testid='loginSubmit']")));
```

Comment:
It would be better move this wait into the method with name "waitElementToBeClickable"and place it in "Base" class and then you can use it in other classes without duplicating wait.until for each element.

Reply:
done